### ISSUE TITLE
Parser doesn't find the texture

### ISSUE THREAD

**Issue by nachtgold** [greenmoss/PyWavefront#11]
I have downloaded `pyglet_demo.py` and its ressources `terran.png`, `uv_sphere.mtl` and `uv_sphere.obj`.

I use `Python 2.7.11 (v2.7.11:6d1b6a68f775, Dec  5 2015, 20:32:19) [MSC v.1500 32 bit (Intel)] on win32` with the latest version of your lib 0.1.2. 

When I run the demo, it prints a long stacktrace, but essential message is:

```
  File "C:\Python27\lib\site-packages\pyglet\resource.py", line 414, in file
    raise ResourceNotFoundException(name)
pyglet.resource.ResourceNotFoundException: Resource "/terran.png" was not found on the path.  Ensure that the filename has the correct captialisation.
```

I reviewed your `parser.py` at line 60:

```
        for arg in args:
            if dir and 'mtllib' in line or 'map_Kd' in line:
                args[i] = dir + '/' + arg.decode("utf-8")
            else:
                args[i] = arg.decode("utf-8")
            i += 1
```

After I added round brackets around the both `in line` checks, the code worked.

`            if dir and ('mtllib' in line or 'map_Kd' in line):`

Does this fit your test cases?

**Comment by greenmoss** [comment]
Try it with the tests, let me know :)

**Comment by jerekshoe** [comment]
There is an existing issue for this [#6](https://github.com/greenmoss/PyWavefront/issues/6). The changes you had to make are already reflected in the current version of the code on Github, so it looks like the latest code has not been pushed to PyPI. In the meantime, the latest version of the code can be downloaded here.

**Comment by greenmoss** [comment]
Thanks @jerekshoe. Released 0.1.3, and uploaded to pypi.
